README.md

```diff
@@ -74,8 +74,8 @@ Before submitting, you must pass all the unit tests and syntax checks by running
 
 ## Notice
 
-* Multiple locales can be built by using comma separated values in the `DEFAULT_LOCALE` variable. i.e: `DEFAULT_LOCALE=en,es,it`.
-* For other 'Serve/Build Options' and 'Test Options', please see `package.json` in details.
+* Multiple locales can be built by using comma-separated values in the `DEFAULT_LOCALE` variable. i.e: `DEFAULT_LOCALE=en,es,it`.
```

Comment:
Optional nit: `e.g.` is what is really meant here, rather than `i.e.`. But since it's obviously an example, it can just be dropped entirely:

```suggestion
* Multiple locales can be built by using comma-separated values in the `DEFAULT_LOCALE` variable: `DEFAULT_LOCALE=en,es,it`.
```